ui/app/templates/components/string-list.hbs

```diff
@@ -9,9 +9,10 @@
   </label>
 {{/if}}
 {{#if warning}}
-  {{#message-in-page type="warning"}}
-    {{warning}}
-  {{/message-in-page}}
+  <AlertBanner
+    @type="warning"
+    @message="{{warning}}"
```

Comment:
This won't eval the string so we'll end up with `"{{warning}}"` in the output.

Reply:
nm this is fine - it's HTML in the strings we can't have unless we do `{{{message}}}` in AlertBanner